compiler+runtime/src/cpp/jank/runtime/obj/big_decimal.cpp

```diff
@@ -124,6 +124,26 @@ namespace jank::runtime::obj
   {
   }
 
+  big_decimal::big_decimal(native_big_integer const &val)
+    : data{ val.real() }
+  {
+  }
+
+  big_decimal::big_decimal(native_big_integer &&val)
+    : data{ val.real() }
+  {
+  }
```

Comment:
There is no point in taking an rvalue reference if there's nothing to move and if you don't even call `std::move`. Both of these rvalue ref ctors should not exist.